admin-part/src/main/kotlin/Session.kt

```diff
@@ -65,7 +61,7 @@ class ActiveSession(
         probe.id?.let { probe } ?: probe.copy(id = probe.id())
     }.forEach { probe ->
         if (true in probe.probes) {
-            val typedTest = probe.testName.weakIntern().typedTest(testType.weakIntern())
+            val typedTest = probe.testName.typedTest(testType)
```

Comment:
type will be repeated, why did you remove testType.weakIntern()?

Reply:
we use .weakIntern in ``String.typedTest(String)```  method